tests/tensor/test_extra_ops.py

```diff
@@ -312,6 +310,7 @@ def test_diffOp(self):
                 g = aesara.function([x], diff(x, n=n, axis=axis))
                 assert np.allclose(np.diff(a, n=n, axis=axis), g(a))
 
+    @pytest.mark.xfail(reason="Subtensor shape cannot be inferred correctly")
```

Comment:
FYI: It should be possible to set `xfail` on only the parameterized test cases that fail (e.g. use something like `pytest.param(..., mark=pytest.mark.xfail(reason=...))`).

Reply:
Thanks for this suggestion, I didn't know that we can selectively mark tests as `xfail`